src/index.ts

```diff
@@ -1,6 +1,6 @@
 import { parseOptions } from './Options';
 import * as Suite from './Suite';
-import * as SpecFormat from './formats/Spec';
+import { findFormat } from './formats';
```

Comment:
question for my own understanding: is this because you're anticipating exporting more functions? otherwise, it's fine to import directly from `findFormat`?

Reply:
Ah, do you mean why are we importing `findFormat` from `./formats` instead of `./formats/findFormat`?

That's just being extra fancy to make the import a little shorter. Maybe it would be better to directly import the file, instead.

Reply:
yep! just curious if there was a strong opinion, I don't have one haha

Reply:
I think it doesn't really matter either way, but I'm going to change it to import directly from the file. Thanks for questioning this.